html_xblock/bleaching.py

```diff
@@ -2,16 +2,7 @@
 A new HTML XBlock that is designed with security and embedding in mind.
```

Comment:
Please bump the version in [setup.py](https://github.com/open-craft/xblock-html/blob/4b7828b/setup.py#L26) to `1.3.0`.